phpcs.ruleset.xml

```diff
@@ -10,5 +10,7 @@
 	<rule ref="vendor/humanmade/coding-standards">
 		<!-- Exclude rules here using the following syntax: -->
 		<!-- <exclude name="HM.Debug.ForceShortArray" /> -->
+		<exclude name="PSR1.Files.SideEffects.FoundWithSymbols" />
```

Comment:
I would prefer not excluding this rule. What's triggering it? (Is it the constants being defined in the main plugin file?)

Reply:
This one stumped me for sure, and I wasn't sure how to resolve. Here's the error I was getting: https://github.com/pantheon-systems/pantheon-wordpress-edge-integrations/runs/5029794551?check_suite_focus=true

(file is now named pantheon-wordpress-edge-integrations.php)

Reply:
Yeah, so it's what I thought. The sniff is saying that constants shouldn't be defined in the same file as code that we're executing. This is probably stricter than we need to be but I've been using these standards for so long at HM that it's what I'm used to. The idea is that a file can _either_ do stuff, or it can declare stuff, but it _can't do both_. We are _declaring things_ by defining globals and variables and we are also _doing things_ by requiring the autoload file and calling the `add_action` function.

You can move those definitions inside `Pantheon\EI\WP\bootstrap()` and you should be all good. (Note, doing so will mean you'll need to tweak the paths to the two constants since you'll be inside the `inc` directory.

Reply:
For reference (scroll down to 2.3): https://www.php-fig.org/psr/psr-1/

Reply:
[8c55797](https://github.com/pantheon-systems/pantheon-wordpress-edge-integrations/pull/8/commits/8c55797130e795f36944495e55f79d439f350466)